core/shared/src/main/scala/org/typelevel/log4cats/LoggerFactory.scala

```diff
@@ -28,4 +32,18 @@ trait LoggerFactory[F[_]] extends LoggerFactoryGen[F] {
 
 object LoggerFactory extends LoggerFactoryGenCompanion {
   def apply[F[_]: LoggerFactory]: LoggerFactory[F] = implicitly
+
+  def mapK[F[_]: Functor, G[_]](fk: F ~> G)(lfg: LoggerFactory[F]): LoggerFactory[G] =
```

Comment:
I'd make this method private and add it to the `LoggerFactory` directly (just like in `SelfAwareStructuredLogger`). What do you think?

Reply:
Yep, good suggestion. Added!